src/integrationtest/docker/tomcat/conf.yml

```diff
@@ -0,0 +1,20 @@
+packaging:
+  add:
+    - target/addon-self-administration-${project.version}
```

Comment:
You can still change the classloader config of Tomcat, like it's explained in the docs. But if it's a big change, the current solution will be ok, I think. The Spring Boot plug-in should normally work and not break the `.war`.